docs/linux-setup.md

```diff
@@ -27,77 +62,128 @@ The following instructions will guide you through the process of installing the
     sudo ufw allow 443
     ```
 
-4. Install Yarn:
+
+3. Install nginx:
     ```
-    sudo snap install yarn --classic
+    sudo apt install nginx -y
     ```
 
-5. Install NVM (Node Version Manager):
+
+4. Install git:
+    ```
+    sudo apt install git -y
+    ```
+
+
+5. Install Docker:
+    ```
+    sudo apt install docker-compose -y
+    ```
+    - Test Docker installation:
+        ```
+        sudo docker run hello-world
+        ```
+
+
+6. Install Certbot:
+    ```
+    sudo apt install certbot -y
+    ```
+
+
+### Suggested packages
+
+These packages are not required to run a node in its basic form. However, they will become necessary for more advanced usage or customization.
+
+
+7. Install ssh (optional, for remote access to your Linux machine):
+    ```
+    sudo apt install openssh-server -y
+    sudo systemctl enable ssh
+    ```
+
+
+8. Install Yarn:
+    ```
+    curl -sSL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+
+    sudo apt-get update -y
+
+    sudo apt-get install yarn -y
+    ```
+
+
+9. Install NVM (Node Version Manager):
     ```
     curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
     source ~/.bashrc
     ```
 
-6. Install Node.js:
+
+10. Install Node.js:
     ```
     nvm install 16.15.1
     ```
 
-7. Install nginx:
-    ```
-    sudo apt install nginx
-    ```
 
-8. Install git:
+11. Install GCC:
```

Comment:
This could simplified. The `build-essential` package includes gcc, g++, and make.